delocate/libsana.py

```diff
@@ -243,6 +244,140 @@ def walk_directory(
                 yield library_path
 
 
+def _tree_libs_from_libraries(
+    libraries: Iterable[str],
+    *,
+    lib_filt_func: Callable[[str], bool],
+    copy_filt_func: Callable[[str], bool],
+    executable_path: Optional[str] = None,
+    ignore_missing: bool = False,
+) -> Dict[str, Dict[str, str]]:
+    """ Return an analysis of the dependencies of ``libraries``.
+
+    libraries : iterable of str
+        The paths to the libraries to find dependencies of.
+    lib_filt_func : callable
+        A callable which accepts filename as argument and returns True if we
+        should inspect the file or False otherwise.
+        If `filt_func` filters a library it will will not further analyze any
+        of that library's dependencies.
+    copy_filt_func : None or callable
```

Comment:
Comment on what `None` means?  Should it be the default?

Reply:
This was a slip up.  This function parameter does not allow None.